data-model/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM nginx:stable-alpine3.17-slim
+
+COPY ./nginx/nginx.conf /etc/nginx/nginx.conf
+
+COPY ./annotations/0.1.0/schema/annotation.json schema-root/schemas/annotations/0.1.0/annotation.json
+
+COPY ./fdo-profiles/0.1.0/digital-specimen/schema/ schema-root/schemas/fdo-profiles/0.1.0/
+COPY ./fdo-profiles/0.1.0/doi-kernel/schema/ schema-root/schemas/fdo-profiles/0.1.0/
+COPY ./fdo-profiles/0.1.0/handle-kernel/schema/ schema-root/schemas/fdo-profiles/0.1.0/
+
+COPY ./specimens-and-media/0.1.0/digital-specimens/schema schema-root/schemas/digital-specimens/0.1.0/
```

Comment:
(See comment on versioning for the beginning of my slow descent into madness) 

Specimens, media, and shared models are intrinsically linked, and I think it makes sense for their versioning to be under one folder. If something in shared-models changes, the references in digital-entity and the specimen schemas would need to be updated. 

Separate directories:

```
| --- specimens 
    | --- 0.1.0
         | --- occurrences.json
         | --- identifications.json
    | --- 0.2.0 
         | --- occurrences.json
         | --- identifications.json
| --- media
     | --- 0.1.0
         | --- digital-entity.json
     | --- 0.2.0 
         | --- digital-entity.json                            
| --- shared-models
     | --- 0.1.0
         | --- agent.json
     | --- 0.2.0 
         | --- agent.json
```

If we group these three directories under one release, it looks cleaner (or as clean as it can): 

```
| --- specimens-and-media
   | --- 0.1.0
      | --- specimens
         | --- occurrences.json
         | --- identifications.json
      | --- media
         | --- digital-entity.json
      | --- shared-models
          | --- agent.jsons
   | --- 0.2.0
      | --- specimens
         | --- occurrences.json
         | --- identifications.json
      | --- media
         | --- digital-entity.json
      | --- shared-models
          | --- agent.json
```
It's also easier in making sure nothing is left behind in a version release.

Reply:
The trouble is, if we update agent.json, we're inadvertently updating both media and specimen models. I think the file system should reflect this.

Reply:
I have been thinking about this A LOT over the last few days.

Reply:
Ok ok, yes and a better name than specimen-and-media? It doesn't look that great in the url. Can we call it `digitalobject` or something, not perfect (as everything is a digitalobject) but shorter and not with the `-and-` in between there?

Reply:
done, also updated the references in the jsonschemas